packages/elements/src/components/ino-radio-group/ino-radio-group.tsx

```diff
@@ -162,10 +169,29 @@ export class RadioGroup implements ComponentInterface {
     return Promise.all(Array.from(this.el.querySelectorAll('ino-radio')));
   }
 
+  private setAlignment() {
+    let style: string;
+    // set default scss class if property is undefined
+    if (!this.alignment) {
+      style = classNames('ino-radio-group-wrapper');
+      return style;
+    }
+
+    switch (this.alignment) {
+      case 'horizontal':
+        style = classNames('ino-radio-group-wrapper');
+        break;
+      case 'vertical':
+        style = classNames('ino-radio-group-wrapper-vertical');
+        break;
+    }
+    return style;
+  }
+
   render() {
     return (
       <Host>
-        <div class="ino-radio-group-wrapper" tabIndex={0}>
+        <div class={this.setAlignment()} tabIndex={0}>
```

Comment:
You can use `classNames` to implement conditional classes like this:
```tsx
const classes = classNames({
	'ino-radio-group-wrapper': true,
  'ino-radio-group-wrapper--vertical': this.alignment === 'vertical',
	'ino-radio-group-wrapper--horizontal': this.alignment === 'horizontal',
})

// ...

	<div class={classes} tabIndex={0}>
```

And I would always set the `ino-radio-group-wrapper` as the default class.